integration_tests/run_pyspark_from_build.sh

```diff
@@ -80,7 +78,7 @@ else
     RUN_DIR="$SCRIPTPATH"/target/run_dir
     mkdir -p "$RUN_DIR"
     cd "$RUN_DIR"
-    if [[ "${FIND_SPARK}" == "1" ]];
+    if [[ "${TEST_PARALLEL}" != "" ]];
```

Comment:
This is here just to make debugging simpler. I found out that with xdist enabled the spark output would not show up on stdout/stderr.  So if you set the parallelism to 1 we run in a way that will output the data.

Reply:
Shouldn't this be:
```suggestion
    if [[ "${TEST_PARALLEL}" != "0" ]];
```
in case someone set it explicitly (e.g.: TEST_PARALLEL=4)